app/main/index.js

```diff
@@ -151,6 +151,54 @@ app.on('ready', () => {
 
 	const page = mainWindow.webContents;
 
+	const ses = page.session;
+
+	const resolveProxyUrl = 'chat.zulip.org';
```

Comment:
@abhigyank is this hardcorded?

Reply:
The main aim is make a request to a website using the mainWindow that can access the system proxy. This `resolveProxyUrl` can be anything, just about any website. Here I have taken `chat.zulip.org` but it could be `google.com` as well.